mlp/neuralnetwork.py

```diff
@@ -58,9 +60,17 @@ def initialize_weights(self, i, j):
         :param j: number of columns
         :return: two-dimensional numpy array
         """
-        # todo why max to max?
-        max_val = 1 / np.sqrt(j)
-        return np.array([[random_uniform(-max_val, max_val) for _ in range(j)] for _ in range(i)])
+        if self.param_init == 'working':
+            max_val = 1 / np.sqrt(j)
+            return np.array([[random_uniform(-max_val, max_val) for _ in range(j)] for _ in range(i)])
+        elif self.param_init == 'zeros':
+            weights = np.zeros((i, j))
+        elif self.param_init == 'normal':
```

Comment:
@ggegiya1 when choosing 'normal' i have errors both in the test run and the full 'mnist' run. I will try to figure it out myself now. The rest works